### PR TITLE
[7.0] Added module 'partner_address_street_3'

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Backport from [8.0].

### CODE REVIEW

File: partner_address_street3/model/partner.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    _inherit = "res.partner"
+    _columns = {
+        'street3': fields.char('Street 3'),
+    }
+
+    def _address_fields(self, cr, uid, context=None):
+        """ Returns the list of address fields that are synced from the parent
+        when the `use_parent_address` flag is set. """
+        return list(ADDRESS_FIELDS)
```
Comment (pedrobaeza) [inline partner_address_street3/model/partner.py]: Why don't you use super here to allow other fields from other modules to be added?

**Comment by JordiBForgeFlow** [comment]
Corrected. Thanks @pedrobaeza!

File: partner_address_street3/model/partner.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+    _inherit = 'res.country'
+
+    _defaults = {
+        'address_format': ("%(street)s\n%(street2)s\n%(street3)s\n"
+                           "%(city)s %(state_code)s %(zip)s\n"
+                           "%(country_name)s"),
+    }
+
+
```
Comment (bwrsandman) [inline partner_address_street3/model/partner.py:60]: From Travis: W391 blank line at end of file

File: partner_address_street3/model/partner.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    def _address_fields(self, cr, uid, context=None):
+        """ Returns the list of address fields that are synced from the parent
+        when the `use_parent_address` flag is set. """
+        res = super(res_partner, self
+                    )._address_fields(cr, uid, context=context)
+        res.append('street3')
+        return res
+
+
```
Comment (bwrsandman) [inline partner_address_street3/model/partner.py:41]: From Travis: E303 too many blank lines (2)

**Comment by gurneyalex** [comment]
overridden by #126

**Pull request closed without merging**